camus-etl-kafka/src/test/java/com/linkedin/camus/etl/kafka/CamusJobTestWithMock.java

```diff
@@ -193,25 +208,117 @@ private void setupJob1() {
     ByteBufferMessageSet messageSet = new ByteBufferMessageSet(messages);
     EasyMock.expect(fetchResponse.messageSet(EasyMock.anyString(), EasyMock.anyInt())).andReturn(messageSet).times(1);
     _mocks.add(fetchResponse);
-    
-    // For SimpleConsumer
-    _simpleConsumer = EasyMock.createMock(SimpleConsumer.class);
+    return fetchResponse;
+  }
+
+  private void mockSimpleConsumer(TopicMetadataResponse metadataResponse, OffsetResponse offsetResponse,
+      FetchResponse fetchResponse) {
+    _simpleConsumer = EasyMock.createNiceMock(SimpleConsumer.class);
```

Comment:
I prefer the createMock because it is more strict.  Can we just add consumer.host(), consumer.port() in the mocked consumer object?